src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -396,7 +409,14 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         volumes.add(new VolumeBuilder().withName(WORKSPACE_VOLUME_NAME).withNewEmptyDir("").build());
         Map<String, Container> containers = new HashMap<>();
 
+        if (template.isCustomJnlpContainerEnabled() && template.getJnlpContainer() != null) {
```

Comment:
this config variable feels misnamed to me.  

the behavior is "when true, automatically add a default jnlp container"

so i would expect a name more like either:

1) IsDefaultJNLPContainerEnabled 
2) IsAddJNLPContainerEnabled

or flip the boolean because

IsCustomJNLPContainerEnabled, to me, means "the user has promised to provide the jnlp container in one way or another, so we don't need to do anything"